src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
+$day-size: 100px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  width: $day-size * 7 + 20px;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    border: 1px solid black;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
It's better to specify to what properties you need to apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/e199b319-e09b-4b21-90c6-fe9bba89d94a)
Also, the cursor move cursor: pointer; to general selector instead of :hover